addons/pandora/model/type.gd

```diff
@@ -60,6 +60,8 @@ func allow_nesting() -> bool:
 static func lookup(name:String) -> PandoraPropertyType:
 	if name == "":
 		return UndefinedType.new()
+	if not ResourceLoader.exists("res://addons/pandora/model/types/" + name + ".gd"):
```

Comment:
we should avoid using absolute paths, can you make this a relative lookup? (in other places too if applicable)

In case this turns out to be difficult, I am happy to keep it absolute for now.

Reply:
I looked into it and unfortunately it doesn't seem possible for the moment (at least not trivially).

* `DirAccess.open` requires an absolute path: "Static methods only support absolute paths (including res:// and user://)." [docs](https://docs.godotengine.org/en/stable/classes/class_diraccess.html#description)
* `load` also requires an absolute path: "Important: The path must be absolute. A relative path will always return null." [docs](https://docs.godotengine.org/en/stable/classes/class_@gdscript.html#class-gdscript-method-load)
* `ResourceLoader.load` doesn't specify but after testing it seems to be the case as well.

It seems that `lookup` and `get_all_types` are implicitly covered by some existing tests (because I broke them when trying) but I could add some tests for `PandoraPropertyType`.

Reply:
The best way to do this is by taking the resource path of the existing class and using it as the root dir for all loading.

ie.

```gdscript
        var klass = PandoraPropertyType
	var types_dir = klass.resource_path.get_base_dir()
	print(types_dir)
	var script_type = types_dir + "/" + name + ".gd"
	print(script_type)
```

Prints
res://addons/pandora/model
res://addons/pandora/model/int.gd